bentoml/configuration/containers.py

```diff
@@ -47,9 +49,10 @@
         },
         "yatai": {"url": Or(str, None)},
         "tracing": {
-            "zipkin_api_url": Or(str, None),
+            Optional("zipkin_server_address"): Or(str, None),
```

Comment:
Why `zipkin_server_address` and port rather than `zipkin_api_url`?
zipkin service is a REST HTTP service, `zipkin_server_address` and `zipkin_server_port` can not represent the API.
For example, `http://127.0.0.1:8080/zipkin/v1/trace`

Reply:
@bojiang what is the convention used in zipkin's documentation & community?  I tried to follow zipkin quickstart guide and setup it with BentoML, it seems to me that specifying an address and host is a bit easier for the user since the user don't need to worry about whether BentoML is using API v1 or v2

Reply:
I looked at a few other Zipkin libraries and it seems like a URL is more often used, I guess we just need to better document this and ask users to provide a full URL like `http://zipkinhost:9411/api/v2/spans`

Reply:
> I looked at a few other Zipkin libraries and it seems like a URL is more often used

It makes sense because zipkin is using HTTP as the transport protocol. (host, port) pair is for L4 protocols.

Reply:
> it seems to me that specifying an address and host is a bit easier for the user since the user don't need to worry about whether BentoML is using API v1 or v2

But the `path` won't be guaranteed by zipkin implements, for example, jaeger. There are no specs that claim a zinkin server has to listen on a specific `path`.

Reply:
@bojiang depends if we want to target the use of default zipkin server https://github.com/openzipkin/zipkin/tree/master/zipkin-server or custom zipkin collector that implements this endpoint

I think the default zipkin server is more commonly used although the other approach is a bit more flexible